<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GoToDialog</class>
 <widget class="QDialog" name="GoToDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Go To Coordinates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLabel" name="rangeY">
     <property name="text">
      <string>Min: X Max: X</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelX_4">
       <property name="text">
        <string>Pos Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="posY">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelX_2">
       <property name="text">
        <string>Cell X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="cellX">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelX">
       <property name="text">
        <string>World X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="worldX">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelY">
       <property name="text">
        <string>World Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="worldY">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="rangeX">
     <property name="text">
      <string>Min: X Max: X</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelY_2">
       <property name="text">
        <string>Cell Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="cellY">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <item>
      <widget class="QLabel" name="labelX_3">
       <property name="text">
        <string>Pos X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="posX">
       <property name="maximum">
        <number>300000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>worldX</tabstop>
  <tabstop>worldY</tabstop>
  <tabstop>cellX</tabstop>
  <tabstop>cellY</tabstop>
  <tabstop>posX</tabstop>
  <tabstop>posY</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GoToDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GoToDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
